lua/dyn_menu.lua

```diff
@@ -13,9 +13,16 @@
 --   #@editions:             edition list
 --   #@audio-devices:        audio device list
 
+local opts = require("mp.options")
 local menu_prop = 'user-data/menu/items'
 local menu_items = mp.get_property_native(menu_prop, {})
 
+local o = {
+    --slice long titles with specified string length
+    slice_longtitles_amount = 80,
+}
+opts.read_options(o)
```

Comment:
```suggestion
opts.read_options(o, 'menu')
```

Reply:
Incorrect suggestions can lead to the following issues:
```log
[w][dyn_menu] script-opts/menu.conf:3 unknown key 'uosc', ignoring 
```

Reply:
Then you will need to configure it with `dyn_menu.conf`. I want to use the same file name for both the DLL and lua script.

To remove this warning, you may add the `uosc` option here with a comment saying it is used by the cplugin.

Reply:
I don't like this solution, I prefer to keep different configuration files.